ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -319,17 +321,12 @@ objects:
 
           if [[ "${rc}" == "1" ]]; then exit 1; fi
 
-          echo "Removing ssh-bastion"
```

Comment:
It needs to be removed before the tests start, otherwise e2e test would fail - we're grabbing image from outside of the release image

Reply:
The whole point of this PR is to run the tests with the bastion running.

Reply:
Right, in that case we have to use some other images in eparis/ssh-bastion repo. See https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/3595/rehearse-3595-pull-ci-openshift-installer-master-e2e-restore-cluster-state/67, test `[Feature:Platform][Smoke] Managed cluster should ensure pods use images from our release image with proper ImagePullPolicy [Suite:openshift/conformance/parallel]` would fail with `fail [github.com/openshift/origin/test/extended/operators/images.go:112]: May 10 09:33:15.658: Pods found with invalid container images not present in release payload: openshift-ssh-bastion/ssh-bastion-f4d5bbcbd-xcx68/ssh-bastion image=quay.io/eparis/ssh:latest`

Reply:
Indeed, that one is going to fail. I think the test could skip `openshift-ssh-bastion` namespace if $KUBE_SSH_BASTION is set.

https://github.com/openshift/origin/pull/23252

Reply:
Later on we shoud be vendoring the manifests in bindata and run it in `ssh-bastion` namespace in BeforeAll

Once https://github.com/openshift/origin/pull/23208 merged I could take care of updating this